pandas/tests/reshape/test_reshape.py

```diff
@@ -608,6 +608,70 @@ def test_get_dummies_all_sparse(self):
         )
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "values",
+        [
+            ["baz", "zoo"],
+            np.array(["baz", "zoo"]),
+            pd.Series(["baz", "zoo"]),
+            pd.Index(["baz", "zoo"]),
+        ],
+    )
+    @pytest.mark.parametrize("method", [True])
+    def test_get_dummies_with_list_like_values(self, values, method):
+        # issue #17160
+        df = pd.DataFrame(
+            {
+                "bar": [1, 2, 3, 4, 5, 6],
+                "foo": ["one", "one", "one", "two", "two", "two"],
+                "baz": ["A", "B", "C", "A", "B", "C"],
+                "zoo": ["x", "y", "z", "q", "w", "t"],
+            }
+        )
+
+        if method:
+            result = pd.get_dummies(df, columns=values, dtype="int64")
+        else:
+            result = pd.get_dummies(df, columns=values, dtype="int64")
+
+        data = [[1, 'one', 1, 0, 0, 0, 0, 0, 1, 0, 0],
```

Comment:
I think `black` is going to want double-quotes on everything here

Reply:
@jbrockmendel I have made changes in my branch. It will reflect in the pull request in some time. Do you have any comments on last point to while creating a pull request (i.e. whatsnew entry)?